js/options.js

```diff
@@ -34,6 +35,17 @@ $(function() {
     loadTenantList();
 });
 
+function saveActiveTenantAndUpdateStatus(tenant) {
+    saveActiveTenant(tenant, function() {
+        // Update status to let user know options were saved.
+        $('#status').html('<div class="alert alert-success">' + chromeOrBrowser().i18n.getMessage('optionsSettingsSaved') + '</div>');
+        $('#optionsHelp').addClass('hidden');
+        setTimeout(function() {
+            $('#status').textContent = '';
```

Comment:
I'm not entirely sure if this is available in the bootstrap that's available in the extension, but, if possible, I would remove this via BS: http://bootstrapdocs.com/v3.3.6/docs/javascript/#alertclose

Reply:
I did briefly look at that and it didn't work. Also I haven't changed this code, simply moved it into a function so I could use it in two places.